### PR TITLE
This is my contribution to your code

### PR DESCRIPTION
**Pull request by Ritikraja07**

### CODE REVIEW

**Comment by blakadder** [comment]
can you elaborate on your reasoning?

**Comment by Ritikraja07** [comment]
yes, sure i am helpful to you if you accept my pull request
I can contribute in your another project also if you need my help
Please accept my pull request

**Comment by blakadder** [comment]
you need to explain your definition of "helpful" since this is not

**Comment by Ritikraja07** [comment]
Helpful means I can help you in other projects too

**Comment by Ritikraja07** [comment]
I can help in solving live projects and real life problems

File: page.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+layout: default
+title: Add New Template
+---
+<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSeCVtKdQLJOejztZTjzTHHGLY26dqpkZ_GySjpouO91VIYjYA/viewform?usp=sf_link" width="760" height="800"  frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
```
Comment (blakadder) [inline page.html:5]: stay in school m8, you'll benefit from it

**Pull request closed without merging**